modules/videoio/src/videoio_registry.cpp

```diff
@@ -45,9 +45,9 @@ namespace {
 #define DECLARE_DYNAMIC_BACKEND(cap, name, mode)  /* nothing */
 #endif
 
-#define DECLARE_STATIC_BACKEND(cap, name, mode, createCaptureFile, createCaptureCamera, createWriter) \
+#define DECLARE_STATIC_BACKEND(cap, name, mode, createCaptureFile, createCaptureCamera, createCaptureBuffer, createWriter) \
```

Comment:
Please add new macro instead of massive refactoring:

`DECLARE_STATIC_BACKEND_WITH_STREAM_SUPPORT`